subfun/expDesign.m

```diff
@@ -48,11 +47,10 @@
 
     % Set variables here for a dummy test of this function
     if nargin < 1 || isempty(cfg)
-        cfg.names             = {'static', 'motion'};
-        cfg.numRepetitions    = 4;
-        cfg.speedEvent        = 4;
-        cfg.numEventsPerBlock = 12;
-        cfg.maxNumFixationTargetPerBlock = 2;
+        cfg.design.names             = {'static', 'motion'};
+        cfg.design.numRepetitions    = 4;
```

Comment:
For consistency I think I tried to switch all the `num` prefix to `nb` everywhere. 
For example, the lines below has `nb` and `Nb`.

But that migth require changes in other places then... Ah the joy of code cleaning... Sometimes if feels like a game of jenga tower.

Reply:
good catch, I missed that. fixed